packages/components/src/core/Callout/style.ts

```diff
@@ -39,12 +38,13 @@ export const StyledCallout = styled(Alert, {
     // any buttom margin
     const titleBottomMargin = props.collapsed ? "margin-bottom: 0;" : "";
 
+    // TODO: Theme shouldn't be saying it might be null
```

Comment:
Type issue, needs to be addressed.

Reply:
So this TODO is a little right and a little wrong, looking at it again.

The `props` here are listed as `CalloutProps`, which extends from the `CommonThemeProps` where `theme` is undefined.

At runtime, `theme` would never be undefined, so that's why I dropped the initial TODO here.

I'm sure we could do a bit of type juggling we could do to make it defined, but I think we'd want to do that in other components that are extending `CommonThemeProps` directly.

Given this PR is more targeted at making dark mode work, I'm inclined to live with the untrue `?` on `theme` for now.